### DISCUSSION TITLE
Welcome to marketing Discussions!

### DISCUSSION THREAD

**Discussion by Truthans** [unikubehq/marketing#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Schille** [comment]
Hi. I am Michael.
If you ever need help integrating unikube in your product development just drop me a line. I am happy to help with all kinds of technical aspects and if you want to talk about challenges and feature requests.
:heart:
